Kickstarter-iOS/Views/Controllers/SettingsAccountViewController.swift

```diff
@@ -122,8 +122,12 @@ final class SettingsAccountViewController: UIViewController, MessageBannerViewCo
   }
 
   private func dismissCurrencyPickerCell() {
+    guard let pickerCell = self.dataSource.removeCurrencyPickerRow() else {
```

Comment:
Can I suggest that we call this `pickerCellIndexPath` just so it's clearer what's being returned.
```suggestion
    guard let pickerCellIndexPath = self.dataSource.removeCurrencyPickerRow() else {
```